src/components/strategies/create/CreateStrategyOption.tsx

```diff
@@ -99,17 +99,21 @@ export const CreateStrategyOption: FC<Props> = ({ base, quote }) => {
                 aria-selected={id === selectedId}
                 onClick={() => setItem(id)}
                 className={cn(
-                  'rounded-10 text-14 flex size-full flex-col items-center justify-start gap-8 bg-black px-8 py-16 outline-white/60',
+                  'rounded-10 text-14 group flex size-full flex-col items-center justify-start gap-8 bg-black px-8 py-16 outline-white/60',
                   'md:px-12',
                   'focus-visible:outline focus-visible:outline-1',
-                  id === selectedId ? 'outline outline-1 outline-white' : ''
+                  id === selectedId
```

Comment:
Could you use `aria-selected:outline` instead of using id === selected ?

Reply:
Added